File: AreaTriangulo.scala
import scala.io.StdIn.readLine
object Area TSCX
def main(args: Array[String]) = {
  //Declaracion de Variables y otros
  println("Ejercicio 01: Area Triangulo")
//Datos de Entrada
println ("Ingrese Base")
val b = readLine().toInt
println("Ingrese Altura:")
val h = readLine().toInt
//Proceso
val area=(b*h)/2
//Datos de Salida
println("El area del triangulo es:"+area)
}
}

File: AreaTriangulo.dart
  
importar  'dart: io' ;
void  main () {
// Definir Variables y otros
print ( 'Ejercicio 01: Area Triangulo' );
// Datos de Entrada
print ( "Ingrese Base:" );
var b = int . analizar (stdin. readLineSync ());
print ( "Ingrese Altura:" );
var h = int . analizar (stdin. readLineSync ());
// Proceso
var área = (b * h) / 2 ;
// Datos de Salida
print ( "El Area de Triangulo es: $ area " );
}